homeassistant/components/powerwall/sensor.py

```diff
@@ -42,20 +41,20 @@ async def async_setup_entry(
     assert coordinator is not None
     data: PowerwallData = coordinator.data
     entities: list[
-        PowerWallEnergySensor | PowerWallEnergyDirectionSensor | PowerWallChargeSensor
-    ] = []
+        PowerWallEnergySensor
```

Comment:
We could simplify the typing of the items in the list and just type it as a SensorEntity. We're passing this list to the entity platform helper which doesn't care about further details than that the list contains Entity instances.

Reply:
https://github.com/home-assistant/core/pull/68345